ibm_catalog.json

```
@@ -0,0 +1,311 @@
+{
+  "products": [
+    {
+      "label": "[Add-ons Beta] Cloud automation for Observability Agents",
+      "name": "testing-deploy-arch-ibm-observability-agents",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "logging_monitoring"
+      ],
+      "short_description": "Deploys IBM Cloud Observability agents to a cluster",
+      "long_description": "Solutions that support deploying IBM Cloud Observability agents for logging, monitoring, and activity tracking.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-da/main/images/observability-icon.svg",
+      "flavors": [
+        {
+          "label": "Standard",
```

New naming convention:
```suggestion
          "label": "Baseline",
```